core/src/test/java/org/infinispan/stream/stress/DistributedStreamRehashStressTest.java

```diff
@@ -78,15 +76,16 @@ protected EmbeddedCacheManager addClusterEnabledCacheManager(TransportFlags flag
 
    public void testStressNodesLeavingWhileMultipleCollectors() throws Throwable {
       testStressNodesLeavingWhilePerformingCallable((cache, masterValues, iteration) -> {
+         SerializablePredicate<Map.Entry<Integer, Integer>> predicate = e -> (e.getKey() & 1) == 1;
          Map<Integer, Integer> results = cache.entrySet().stream().filter(
-                 (Serializable & Predicate<Map.Entry<Integer, Integer>>)
-                         e -> (e.getKey() & 1) == 1).collect(
+               predicate).collect(
```

Comment:
Formatting is a bit weird now

Reply:
Oh whoops, not sure how that happened :)